application/view/omeka/admin/job/show.phtml

```diff
@@ -59,9 +59,9 @@ if ($owner = $job->owner()) {
     <?php if ($job->log()): ?>
     <?php echo $this->hyperlink($this->translate('view log'),
         $this->url(null, ['action' => 'log'], true),
-        ['target' => '_blank']) ?>
+        ['target' => '_blank']); ?>
     <?php else: ?>
-    <?php echo $this->translate('[no log]') ?>
+    <?php echo $translate('[no log]'); ?>
```

Comment:
This seems to have been left behind from one of the "convert to $translate" commits.

Reply:
Ok.